test/resources/config/regexp.bal

```diff
@@ -0,0 +1,24 @@
+function name() {
+
+    string pattern = "ABC";
+    
+    string:RegExp _ = re `abc`;
+    string:RegExp _ = re `(abc)`;
+    string:RegExp _ = re `[abc]`;
+
+    string:RegExp _ = re `\p{gc=Ll}|\P{Lm}|\p{sc=Latin}`;
+    string:RegExp _ = re `\u{12FFFF}`;
+
+    string:RegExp _ = re `${pattern}`;
```

Comment:
In here, the colours of the `$` and the Curley brackets are different.
Don't we need to give the same colours for `$` and `{ }`, like we do in the `\p{}`.

Reply:
no. Those two comes under two different scopes and they have different meanings.

`\p{Ll}` and `\u{FFF}` cases should give the same color.